lib/repo/git_repository.rb

```diff
@@ -390,54 +474,75 @@ def get_users(permissions)
     end
 
     def get_permissions(user_id)
+      $stderr.puts "In get perms"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.